ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/block/BlockProcessorElectra.java

```diff
@@ -117,6 +124,29 @@ public BlockProcessorElectra(
     this.beaconStateMutatorsElectra = beaconStateMutators;
     this.beaconStateAccessorsElectra = beaconStateAccessors;
     this.schemaDefinitionsElectra = schemaDefinitions;
+    this.executionRequestsDataCodec =
```

Comment:
I think I prefer adding this initialization in `SpecLogicElectra` for consistency.